inc/views/base/calendar/calendar.twig

```diff
@@ -0,0 +1,105 @@
+                {% endfor %}
+            </tr>
+        </thead>
+        <tbody>
+            {% for week in weeks %}
+                <tr>
+                    <td class="tcat" align="center" width="1"><a href="{{ week.week_link|raw }}">&raquo;</a></td>
+                    {% for day in week.days %}
+                        <td class="{{ day.day_class }}" style="vertical-align: top; height: 100px;">
+                            <div class="float_right smalltext"><a href="{{ day.day_link|raw }}">{{ day.day }}</a></div>
+                            <div class="clear">
+                                {% if day.bday_count > 0 %}
+                                     <div style="margin-bottom: 4px;"><a href="{{ day.calendar_link|raw }}" class="smalltext">{{ day.bday_count }} {{ day.birthday_lang }}</a></div>
+                                {% endif %}
+                                {% if day.eventlimit == true %}
+                                    <div style="margin-bottom: 4px;"><a href="{{ calendar.link }}" class="smalltext">{{ day_bit.total_events }} {{ day_bit.event_lang }}</a></div>
+                                {% else %}
+                                    {% for event in day.events %}
+                                        <div style="margin-bottom: 4px;" class="smalltext {{ event.event_class }}"><a href="{{ event.eventlink|raw }}" title="{{ event.fullname }}">{{ event.name }}</a></div>
+                                    {% endfor %}
+                                {% endif %}
+                            </div>
+                        </td>
+                    {% endfor %}
+                </tr>
+            {% endfor %}
+        </tbody>
+    </table>
+    <br />
+    <form action="calendar.php" method="post">
+        <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+            <tr>
+                <td class="trow1">
+                    <table width="100%" cellspacing="0" cellpadding="0" border="0">
+                        <tr>
+                            <td valign="top">
+                                {% if calendar_permissions.canaddevents %}
+                                    <a href="calendar.php?action=addevent&amp;calendar={{ calendar.cid }}">{{ lang.add_public_event }}</a> | <a href="calendar.php?action=addevent&amp;calendar={{ calendar.cid }}&amp;private=1">{{ lang.add_private_event }}</a>
+                                {% endif %}
+                            </td>
+                            <td align="right">
+                            <span class="smalltext"><strong>{{ lang.jump_month }}</strong></span>
+                            <select name="month">
+                                <option value="{{ calendar.month }}">{{ calendar.currentmonth }}</option>
+                                <option value="{{ calendar.month }}">----------</option>
+                                <option value="1">{{ lang.alt_month_1 }}</option>
```

Comment:
Again, could use the same loop trick I mentioned in the other template file hopefully.